server/graphql/common/transactions.ts

```diff
@@ -121,10 +122,14 @@ export const canRefund = async (
   if (transaction.type !== TransactionTypes.CREDIT || transaction.OrderId === null || transaction.isRefund === true) {
     return false;
   }
+
   const timeLimit = moment().subtract(30, 'd');
   const createdAtMoment = moment(transaction.createdAt);
   const transactionIsOlderThanThirtyDays = createdAtMoment < timeLimit;
-  if (transactionIsOlderThanThirtyDays) {
+  /*
+   * To ensure proper accounting, we only allow added funds to be refunded by Host admins and never by Collective admins.
+   */
```

Comment:
The description is not accurate anymore as it doesn't explain `transactionIsOlderThanThirtyDays`.

Reply:
I've added another line to emphasize that the `transactionIsOlderThanThirtyDays` condition as well. We didn't have a comment about this part before, so I am not exactly sure if there's a reason for this 30 day limitation; if so let me know and I'll update the comment. 🤔